### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by 0xGabi**
# 🦅 Pull Request

Remove `init`, `network` and clean `environment` middleware

Modify global config:
`apm.ens-registry` -> `ens-registry`
`apm.ipfs.rpc` -> `ipfs-rpc`
`apm.ipfs.gateway` -> `ipfs-gateway`

### CODE REVIEW

**Comment by 0xGabi** [comment]
@dapplion would you mind take a look 🙏 sorry it took a bit longer

**Comment by 0xGabi** [comment]
No worries to much on review this, will review and merge with #893

**Comment by dapplion** [comment]
> @dapplion would you mind take a look  sorry it took a bit longer

Sure! Let's brace ourselves for all the conflicts with the refactor branches

File: packages/aragon-cli/src/commands/dao_cmds/acl_cmds/view.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ exports.handler = async function({
               }
             }
 
-            initAragonJS(dao, apm['ens-registry'], {
+            initAragonJS(dao, apm.ensRegistryAddress, {
```
Comment (dapplion) [inline packages/aragon-cli/src/commands/dao_cmds/acl_cmds/view.js]: Is it `apm.ensRegistryAddress` or `apm.ensRegistry`? If it's not an error let's make it consistent for clarity or explain what each variable represents. 
`ensRegistryAddress` is also used in:
- https://github.com/aragon/aragon-cli/pull/895/files#diff-dbe6805e3cab1be07d9137504711cb05R59
- https://github.com/aragon/aragon-cli/pull/895/files#diff-f0a56a91e95462ee3c2a7f43e5ee4974R401
- https://github.com/aragon/aragon-cli/pull/895/files#diff-7fb765944652f71e14673b01c4e751d9R66

File: packages/aragon-cli/src/middleware/environment.js [hunk 2]
```diff
@@ -121,50 +105,32 @@ module.exports = function environmentMiddleware(argv) {
...
+      env.network === 'rinkeby'
+        ? ARAGON_RINKEBY_ENDPOINT
+        : env.network === 'mainnet'
+        ? ARAGON_MAINNET_ENDPOINT
+        : env.wsRPC
+
+    return {
       module: Object.assign({}, module, { appName: env.appName }),
+      apm: configureAPM(
+        env.apm.ipfs.rpc || argv['ipfs-rpc'],
```
Comment (dapplion) [inline packages/aragon-cli/src/middleware/environment.js]: Can you guarantee that `env.apm` is defined and is an object in this line?

File: packages/aragon-cli/src/middleware/environment.js [hunk 3]
```diff
@@ -178,20 +144,12 @@ module.exports = function environmentMiddleware(argv) {
...
-      if (env.apm) {
-        apm = merge(apm, env.apm)
-      }
-    }
-
     return {
-      // TODO (daniel) : remove this as it does not seem to be used
-      apmEnsRegistry: env.registry,
+      apm: configureAPM(
+        env.apm.ipfs.rpc || argv['ipfs-rpc'],
```
Comment (dapplion) [inline packages/aragon-cli/src/middleware/environment.js]: Same as https://github.com/aragon/aragon-cli/pull/895/files#diff-7fb765944652f71e14673b01c4e751d9R118

Can you guarantee that env.apm is defined and is an object in this line?

**Comment by dapplion** [comment]
Also, in multiple commands, the `network` argument coming from the `yargs` object (for example: see below)
https://github.com/aragon/aragon-cli/blob/17fc54172ac3f2556a24391a3388173d5d653481/packages/aragon-cli/src/commands/deploy.js#L133
That network parameter is the one from the middleware or from the `yargs` option?

**Comment by 0xGabi** [comment]
The concern with the `network` is valid, I think we should decide on a better name, I like `web3provider`. Also, I think we should have a provider object with all the providers.

**Comment by 0xGabi** [comment]
Will move the changes to #895

**Pull request closed without merging**